backend/de.metas.shipper.gateway.commons/src/main/java/de/metas/shipper/gateway/commons/ShipperGatewayFacade.java

```diff
@@ -180,4 +183,14 @@ public boolean hasServiceSupport(@NonNull final String shipperGatewayId)
 		return shipperRegistry.hasServiceSupport(shipperGatewayId);
 	}
 
+	private static UOMPrecision getKgUOMPrecision()
+	{
+		if (kgPrecision == null)
+		{
+			final IUOMDAO uomDAO = Services.get(IUOMDAO.class);
```

Comment:
pls make this a field

Reply:
yup, was put off by `de.metas.shipper.gateway.commons.ShipperGatewayFacade#computeGrossWeightInKg` being static, but there's really not reason for it to be static.